src/core/src/package_managers/AptitudePackageManager.py

```diff
@@ -654,6 +654,9 @@ def set_security_esm_package_status(self, operation):
                 if not self.ubuntu_pro_client.is_ubuntu_pro_client_attached:
                     self.status_handler.add_error_to_status("{0} patches requires Ubuntu Pro for Infrastructure with Extended Security Maintenance".format(len(security_esm_updates)), Constants.PatchOperationErrorCodes.UA_ESM_REQUIRED)
             elif operation == Constants.INSTALLATION:
+                if security_esm_update_query_success:
+                    esm_packages_selected_to_install = [package for package in packages if package in security_esm_updates]
+                    self.composite_logger.log_debug("[SelectedEsmPackagesCount={0}]".format(len(esm_packages_selected_to_install)))
```

Comment:
Setting security ESM package status. [Operation=Installation][SelectedEsmPackagesCount={0}]

Reply:
Operation=Installation is not necessary as this scenario will only be there in installation scenario. Will add the rest.

Reply:
Done.